src/components/organization/organization.service.ts

```diff
@@ -118,7 +118,7 @@ export class OrganizationService {
     ];
   };
 
-  async create(
+  async creates(
```

Comment:
did you leave the old version in for a reason?  Please remove this.  If we need the history we can use git history.